docs/examples/Rating.jsx

```diff
@@ -0,0 +1,4 @@
+var myComponentInstance = (
```

Comment:
can we stick to the existing style of naming these instances after their type? e.g. `ratingInstance`